scripts/src/actions/sync-labels.ts

```diff
@@ -0,0 +1,95 @@
+import 'reflect-metadata'
+import {Octokit} from '@octokit/rest'
+import {GitHub} from '../github'
+import env from '../env'
+import * as core from '@actions/core'
+import {GetResponseDataTypeFromEndpointMethod} from '@octokit/types' // eslint-disable-line import/named
+
+const Endpoints = new Octokit()
+type Labels = GetResponseDataTypeFromEndpointMethod<
+  typeof Endpoints.issues.getLabel
+>[]
+
+async function getLabels(repo: string): Promise<Labels> {
+  // initialize GitHub client
+  const github = await GitHub.getGitHub()
+
+  // use the GitHub client to fetch the list of labels from js-libp2p
+  const labels = await github.client.paginate(
+    github.client.issues.listLabelsForRepo,
+    {
+      owner: env.GITHUB_ORG,
+      repo: repo
+    }
+  )
+
+  return labels
+}
+
+async function addLabel(
+  repo: string,
+  name: string,
+  color: string,
+  description: string | undefined
+) {
+  // initialize GitHub client
+  const github = await GitHub.getGitHub()
+
+  await github.client.issues.createLabel({
+    owner: env.GITHUB_ORG,
+    repo: repo,
+    name: name,
+    color: color,
+    description: description
+  })
+}
+
+async function removeLabel(repo: string, name: string) {
+  // initialize GitHub client
+  const github = await GitHub.getGitHub()
+
+  await github.client.issues.deleteLabel({
+    owner: env.GITHUB_ORG,
+    repo: repo,
+    name: name
+  })
+}
+
+async function sync() {
+  const sourceRepo = 'js-libp2p'
+  const targetRepos = ['js-libp2p-peer-store', 'js-libp2p-connection']
```

Comment:
Removing archived repo: https://github.com/libp2p/js-libp2p-connection

```suggestion
  const targetRepos = ['js-libp2p-peer-store']
```